recipes/gsort/meta.yaml

```diff
@@ -0,0 +1,23 @@
+package:
+  name: gsort
+  version: '0.0.1'
+
+source:
+  fn: gsort_darwin_amd64 # [osx]
+  url: https://github.com/brentp/gsort/releases/download/v0.0.1/gsort_darwin_amd64 # [osx]
```

Comment:
These are pre-built static GO binaries so hopefully will work pretty cleanly across most systems. I'm personally a fan of using binaries where it makes sense so am not very strict about trying to build everything from source.